queries/bibtex/highlights.scm

```diff
@@ -0,0 +1,49 @@
+; CREDITS @pfoerster (taken from https://github.com/latex-lsp/tree-sitter-bibtex)
+
+[
+  (string_type)
+  (preamble_type)
+  (entry_type)
+] @keyword
+
+[
+  (junk)
+  (comment)
+] @comment
+
+[
+  "="
+  "#"
+] @operator
+
+(command) @function.builtin
+
+(number) @number
+
+(field
+  name: (identifier) @variable.builtin)
+
+(token
+  (identifier) @variable.parameter)
+
+[
+  (brace_word)
+  (quote_word)
+] @string
+
+[
+  (key_brace)
+  (key_paren)
+] @attribute
```

Comment:
`attribute` has the moment the meaning of `code annotation`. We should probably change that... memo to myself. I would leave it that way.

Reply:
yeah, I don't know the proper highlight groups for tree-sitter, I'm afraid. That's why the checks are failing 😞 

I'd  be grateful for suggestions for appropriate groups!

Reply:
no, attribute exists. The failure is due to `@variable.parameter`

Reply:
I mean "the failure is due to the fact that I don't know what I'm doing, generally" :]